spark/src/main/kotlin/com/adevinta/spark/components/textfields/MultilineTextField.kt

```diff
@@ -246,17 +227,32 @@ public fun MultilineTextField(
     placeholder: String? = null,
     helper: String? = null,
     counter: TextFieldCharacterCounter? = null,
-    leadingIcon: @Composable (() -> Unit)? = null,
-    isError: Boolean = false,
-    error: String? = null,
+    leadingContent: @Composable (() -> Unit)? = null,
+    state: TextFieldState? = null,
+    stateMessage: String? = null,
```

Comment:
it seems that a @param is missing for new params: state and stateMessage

Reply:
hum nope it's actually there 🤔